.github/workflows/review.yaml

```diff
@@ -50,13 +50,65 @@ jobs:
             requirements.txt
             requirements-dev.txt
 
+      - name: Install mewbot-dev-tools
+        run: python -m pip install mewbot-dev-tools
+        shell: sh
+
+      - name: Install dependencies into default python
+        shell: bash
+        run: python -m mewbot_dev_tools.install_deps
+
+      - name: Run Tests (without install)
```

Comment:
- labelling improved
 - matrices have been simplified
 - it's more complicated than I'd like, but I thought splitting the tasks down into seperate lint and test matrices made it more readable
 - I considered doing the path hacking in tools, but it seemed better to do it explicitely in the action
 - Ideally we'd run the tests both ways - after getting mewbot on the path via path hacks and after running setup.py
 - But I wanted to discuss that change before implementing it - so made a minimal set of changes to the workflow - just enough to get it running with the new tools package

Reply:
I've taken a stab at recombining the two in c5f51d1.
The workflow works, but I've noticed a couple of...oddities:

 - Coverage report does not include `src/mewbot/io/desktop_notification.py`. Might be that there's no tests, but I would expect it to have a 0% entry in the table ([ref](https://github.com/mewbotorg/mewbot/actions/runs/6453497714/job/17517174505#step:6:177))
 - non-zero exit code of `istort` isn't being picked up ([ref 1](https://github.com/mewbotorg/mewbot/actions/runs/6453497714/job/17517173968#step:6:32) | [ref 2](https://github.com/mewbotorg/mewbot/actions/runs/6453497714/job/17517173968#step:6:76))
 - At some point, we should tidy up that `arg_list` output in the dev tools

Reply:
Indeed - this'll have to be solved on the mewbot-dev-tools side.

Will fix 2 and 3.

Point 1 seems to boil down to pytest not handling namespace plugins correctly. Will experiment and find a workaround (as I suspect this will happen with any custom namespace package as well ... which is suboptimal)

Reply:
Point 1 haos been traced to a - seeming - issue in how coverage itself handles namespace plugins (it doesn't - not correctly, anyhow).

I have a extremely hacky solution
 - on run time, introspect to generate a new test file in the test tree
 - said test file just contains imports for all the namespace plugins found in mewbot.io
 - this seems to work - but might be a cure worse than the disease

Reply:
2 and 3 have been addressed in mewbot-dev-tools 0.0.13